pkg/controllers/register.go

```diff
@@ -50,6 +52,7 @@ var ResourceControllers = map[string][]func(*synccontext.RegisterContext) (synce
 	"volumesnapshots":        newControllers(volumesnapshotclasses.New, volumesnapshots.New, volumesnapshotcontents.New),
 	"serviceaccounts":        newControllers(serviceaccounts.New),
 	"persistentvolumes,fake-persistentvolumes": newControllers(persistentvolumes.New),
+	"k8sdefaultendpoint":                       newControllers(k8sdefaultendpoint.New),
```

Comment:
I guess this would require a bit of a refactor to the k8sdefaultendpoint controller to make it a generic controller. We should probably discuss this option.